nservicebus/upgrades/5to6/distributor.md

```diff
@@ -0,0 +1,188 @@
+* Run the query.
+* On all listed documents, remove the entry with the Distributor's subscriber address and save it.
+
+
+### Removing subscriptions from NHibernate persistence
+
+Execute the following script against the database which is configured for NHibernate persistence:
+
+```
+DELETE
+FROM dbo.Subscription
+WHERE SubscriberEndpoint = '<distributorAddress>'
+```
+
+where `<distributorAddress>` is the address of the Distributor. E.g. `My.Endpoint@distributor-machine`.
+
+
+### Removing subscriptions from Azure Storage Persistence
+
+The following Powershell script removes all subscriptions of a specific subscriber from the configured Azure storage account. Copy & Paste the script to a Powershell and hit Enter:
+
+```
+function Remove-Subscriptions(
+    [string] $accountStorageName,
+    [string] $accountStorageKey,
+    [string] $subscriptionTableName = 'Subscription',
+    [string] $transportAddressToRemove
+    ) 
+{
+    # create Context
+    $ctx = New-AzureStorageContext -StorageAccountName $accountStorageName -StorageAccountKey  $accountStorageKey
+
+    # get table 
+    $table = Get-AzureStorageTable -Name $subscriptionTableName -Context $ctx -ErrorAction Ignore
+
+    # if exists
+    if ($table -ne $null) {
+
+        #Create a table query.
+        $query = New-Object Microsoft.WindowsAzure.Storage.Table.TableQuery
+
+        #Define columns to select.
+        $list = New-Object System.Collections.Generic.List[string]
+        $list.Add("RowKey")
+        $list.Add("PartitionKey")
+        $list.Add("EndpointName")
+
+        #Calculate RowKey based on addressToRemove
+        $asciiEncoded = [System.Text.Encoding]::ASCII.GetBytes($transportAddressToRemove)
+        $base64Encoded = [System.Convert]::ToBase64String($asciiEncoded)
+
+        #Set query details.
+        $query.FilterString = "RowKey eq '" + $base64Encoded + "'"
+        $query.SelectColumns = $list
+        
+        #Execute the query.
+        $entries = $table.CloudTable.ExecuteQuery($query)
+
+        Write-Host "Following subscriptions will be removed for " $transportAddressToRemove
+
+        $entries | Format-Table @{ Label = "MessageType"; Expression={$_.PartitionKey}}, @{ Label = "EndpointName"; Expression={$_.Properties["EndpointName"].StringValue }} -AutoSize
+
+        $confirmation = Read-Host "Are you sure you want to remove this entries? [y/n]"
+
+        if ($confirmation -eq 'y') {
+            # proceed
+
+            foreach ($entry in $entries) {
+                $table.CloudTable.Execute([Microsoft.WindowsAzure.Storage.Table.TableOperation]::Delete($entry))
+            }
+        } else {
+            Write-Host 'Deletion aborted'
+        }
+    }
+}
+```
+
+Then execute the script:
+
+`Remove-Subscriptions -accountStorageName {storageAccountName} -accountStorageKey {storageAccountKey} -transportAddressToRemove {distributorAddress}`
```

Comment:
I'd consider dropping the brackets around parameter values `{ }`. The most common way of providing examples I've seen so far is to use `your` prefix like:

`-accountStorageName yourStorageAccountName`